src/bundle/ui-dev/src/modules/common/simple-dropdown/simple.dropdown.js

```diff
@@ -0,0 +1,121 @@
+import React, { useState, useEffect, useRef } from 'react';
+import PropTypes from 'prop-types';
+
+import { createCssClassNames } from '../helpers/css.class.names';
+import Icon from '../icon/icon';
+
+const SimpleDropdown = ({ options, selectedOption, extraClasses, onOptionClick, isDisabled, selectedItemLabel }) => {
+    const containerRef = useRef();
+    const [isExpanded, setIsExpanded] = useState(false);
+    const dropdownClass = createCssClassNames({
+        'c-simple-dropdown': true,
+        'c-simple-dropdown--expanded': isExpanded,
+        'c-simple-dropdown--disabled': isDisabled,
+        [extraClasses]: true,
+    });
+    const toggleExpanded = () => {
+        if (isDisabled) {
+            return;
+        }
+
+        setIsExpanded((prevState) => !prevState);
+    };
+    const onOptionClickWrapper = (option) => {
+        onOptionClick(option);
+
+        setIsExpanded(false);
+    };
+    const renderItem = (item) => {
+        const isItemSelected = item.value === selectedOption?.value;
+        const itemClass = createCssClassNames({
+            'c-simple-dropdown__list-item': true,
+            'c-simple-dropdown__list-item--selected': isItemSelected,
+        });
+
+        return (
+            <li key={item.value} className={itemClass} onClick={() => onOptionClickWrapper(item)}>
+                {item.iconName && <Icon name={item.iconName} extraClasses="c-simple-dropdown__list-item-type-icon ibexa-icon--small" />}
+                <span>{item.label}</span>
+                {isItemSelected && <Icon name="checkmark" extraClasses="c-simple-dropdown__list-item-checkmark ibexa-icon--small" />}
+            </li>
+        );
+    };
+    const renderCaretIcon = () => {
+        const iconName = isExpanded ? 'caret-up' : 'caret-down';
+
+        return <Icon name={iconName} extraClasses="ibexa-icon--tiny c-simple-dropdown__expand-icon" />;
+    };
+    const renderSelectedLabel = () => {
+        if (!selectedOption && selectedItemLabel.length) {
+            return null;
+        }
+
+        return (
+            <span className="c-simple-dropdown__selected-item-label">
+                {selectedItemLabel.length ? selectedItemLabel : selectedOption.label}
+            </span>
+        );
+    };
+    const renderSelectedIcon = () => {
+        if (!selectedOption || !selectedOption.iconName) {
```

Comment:
```suggestion
        if (!(selectedOption?.iconName)) {
```
but I'm not sure now if it's more readable, so maybe leave it as it is...

Reply:
solved privately

Reply:
We agreed that the original solution is more readable.